client/my-sites/themes/theme-options.js

```diff
@@ -88,6 +92,16 @@ const tryandcustomize = {
 	hideForTheme: ( state, theme, siteId ) => isActive( state, theme.id, siteId )
 };
 
+const tryAndCustomizeOnJetpack = {
+	label: i18n.translate( 'Try & Customize' ),
+	header: i18n.translate( 'Try & Customize on:', {
+		comment: 'label in the dialog for opening the Customizer with the theme in preview'
+	} ),
+	action: tryAndCustomizeWpcomThemeOnJetpack,
```

Comment:
What about dropping the `installWpcomThemeAndTryAndCustomize` wrapper and doing

```js
action: ( themeId, siteId ) => {
	installAndTryAndCustomize( themeId + '-wpcom', siteId )
}
```

here?